package.json

```diff
@@ -55,16 +55,14 @@
     "node": "^12.22.0 || ^14.17.0 || >=16.0.0"
   },
   "devDependencies": {
-    "@babel/code-frame": "^7.16.0",
-    "@babel/parser": "^7.16.4",
-    "@babel/types": "^7.16.0",
```

Comment:
we still need these for the AST alignment tests I think?

Reply:
Do we? If the test suite passes 🤷

Reply:
https://github.com/typescript-eslint/typescript-eslint/blob/699ef484034b932ae8b2a23575ac4dae552f2886/packages/typescript-estree/package.json#L53-L55

Yeah so right now it'll just install the latest, which we don't want!

The reason we had the dep declared here was because it was used across multiple packages. I don't think it is any more, so we can put these numbers in the ts-estree package.json

Reply:
I did! https://github.com/typescript-eslint/typescript-eslint/pull/4584/files#diff-64a7f8ef3369c510e2a7a43d79350450bfa85109863b698ec377f11717e52559R55